pagespeed/system/system_rewrite_driver_factory.h

```diff
@@ -258,6 +258,23 @@ class SystemRewriteDriverFactory : public RewriteDriverFactory {
     return 1;
   }
 
+  // By default this uses the ControllerManager to fork off some processes to
+  // handle the Controller.  If you're on a system where fork doesn't make
+  // sense or running the Controller in its own process doesn't make sense, this
+  // is a hook where you can start the controller in whatever way makes sense
+  // for your platform.
+  virtual void StartController();
+
+  // Set the name of this process, for debugging visibility.
+  virtual void NameProcess(const char* name);
+
+  // Hook for handling any process-specific initialization the host webserver
+  // might need when we manually fork off a process.
```

Comment:
Mention that the base version should be called by subclasses, and what it does?
(Seems needed due to is_root_process_ assignment)